Projects/App/Sources/UI/Organization/DomainSetting/View/DomainSettingViewController.swift

```diff
@@ -0,0 +1,198 @@
+        emailStackView.alignment = .fill
+
+        emailTextField.becomeFirstResponder()
+        emailTextField.attributedPlaceholder = .init(attributedString: NSAttributedString(string: "이메일", attributes: [NSAttributedString.Key.foregroundColor: UIColor.lightGray]))
+        emailTextField.textColor = .white
+        emailTextField.font = .systemFont(ofSize: 17, weight: .medium)
+        emailTextField.keyboardType = .asciiCapable
+        
+        domainPlaceholder.textColor = .white
+        domainPlaceholder.text = "@pos.idserve.net"
+        domainPlaceholder.font = .systemFont(ofSize: 17, weight: .medium)
+        domainPlaceholder.setContentCompressionResistancePriority(.init(1000), for: .horizontal)
+        
+        emailDuplicatedLabel.textColor = .red
+        emailDuplicatedLabel.text = "이미 사용된 이메일이에요."
+        emailDuplicatedLabel.font = .systemFont(ofSize: 13, weight: .regular)
+        
+        // TODO: Component로 변경
+        arrowButton.tintColor = .lightGray
+        arrowButton.backgroundColor = .white
+        arrowButton.configuration = .plain()
+        arrowButton.configuration?.contentInsets = .init(top: 14.5, leading: 15, bottom: 14.5, trailing: 15)
+        arrowButton.clipsToBounds = true
+        arrowButton.layer.borderWidth = 2
+        arrowButton.layer.cornerRadius = 25
+        arrowButton.layer.borderColor = UIColor.lightGray.cgColor
+        
+        let arrowImageConfiguration = UIImage.SymbolConfiguration(pointSize: 16, weight: .bold, scale: .default)
+        let arrowImage = UIImage(systemName: "arrow.forward", withConfiguration: arrowImageConfiguration)
+        arrowButton.setImage(arrowImage, for: .normal)
+    }
+    
+    private func createLayout() {
+        view.addSubviews([mainTitleView, keyboardSafeArea])
+        
+        keyboardSafeArea.addSubviews([emailInputView, emailDuplicatedLabel, arrowButton])
+        
+        emailInputView.addSubview(emailStackView)
+        emailStackView.addArrangedSubviews([emailTextField, domainPlaceholder])
+        
+        mainTitleView.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide)
+            make.horizontalEdges.equalToSuperview()
+        }
+    }
+    
+    private func updateLayout(keyboardHeight: CGFloat? = nil) {
+        guard let keyboardHeight = keyboardHeight else { return }
+        
+        keyboardSafeArea.snp.makeConstraints { make in
+            make.top.equalTo(mainTitleView.snp.bottom)
+            make.horizontalEdges.equalToSuperview()
+            make.bottom.equalToSuperview().inset(keyboardHeight)
+        }
+        
+        emailInputView.snp.makeConstraints { make in
+            make.centerY.equalToSuperview()
+            make.centerX.equalToSuperview()
+            make.height.equalTo(50)
+            make.width.lessThanOrEqualTo(310)
+        }
+        
+        emailStackView.snp.makeConstraints { make in
+            make.verticalEdges.equalToSuperview().inset(14)
```

Comment:
p3;
emailInputView의 높이가 정해져있으니까 emailStackView의 상하 inset은 줄 필요가 없겠네요 오히려 레이아웃이 깨지는 원인이 될지도...
중앙 정렬만 하면 될 것 같습니다!
```suggestion
            make.centerY.equalToSuperview()
```